Assets/Scripts/BasicMove.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+
+// Moves object in a single direction. Great for testing movement.
+public class BasicMove : MonoBehaviour
+{
+    public Vector3 moveDirection = new Vector3(0, 1, 0);
+    public Vector3 speed = new Vector3(1, 1, 1);
```

Comment:
Nit: speed is a scalar quantity. Would suggest naming it `velocity` unless that's a built-in field name and could cause confusion.

Reply:
Updated, thanks.